web_client/templates/VideoWidget.pug

```diff
@@ -0,0 +1,29 @@
+.g-video-widget
+  .g-first-line
+    span.status
+      if status === 0
```

Comment:
is there a better practice available than using these magic numbers in the code, e.g. 0, 1, 2?  I would prefer at least if these are defined as constants.

Reply:
The status number is coming from Girder Jobs state, I am reusing job state because it looks like the backend will use jobs for video processing. I think make it act like enum will be helpful.

Reply:
Ok, I took a look. It seems it's not possible to refer to JobStatus like an enum at this moment.